COMPATIBILITY.md

```diff
@@ -1,13 +1,17 @@
-# Plugin Compatibility with Workflow
+# Workflow Compatibility Plugins
 
-For architectural reasons, plugins providing various extensions of interest to builds cannot be made automatically compatible with Workflow.
-Typically they require use of some newer APIs, large or small (see the bottom of this document for details).
-This document captures the ongoing status of plugins known to be compatible or incompatible.
+For architectural reasons, plugins that provide various extensions of interest to builds are not automatically compatible with Workflow.
+Typically, they require use of newer APIs - both large or small.
+This document details the ongoing status of plugins known to be compatible or incompatible.
 
-Entries list the class name serving as the entry point to the relevant functionality of the plugin (generally an `@Extension`), the plugin short name, and implementation status.
+Entries list:
+* the class name serving as the entry point to the relevant functionality of the plugin (generally an `@Extension`)
+* the plugin short name  
+* implementation status
 
-## SCMs
+## Compatible SCMs
 
+The following SCMs are compatible. See [this guide](scm-step/README.md#supporting-workflow-from-an-scm-plugin) for further details on making `SCM`s compatible.
```

Comment:
Looks like your Git merge failed, because this is reintroducing a line I intentionally deleted some time ago. Always review the effective diff after a merge for sanity.